woocommerce/woocommerce-ios
# Add and use ListPressableButton

Closes: #11660


## Description

- second part of https://github.com/woocommerce/woocommerce-ios/pull/11777
- Added and using **ListPressableButton** to be able to have a custom SwiftUI List background color when button is pressed

## Testing instructions

- Open Menu part of the app
- Tap on various items in the list and check that the background color is correct and the button actions work properly

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screen Recording - iPhone 15 Pro - 2024-02-01 at 12 44 23](https://github.com/woocommerce/woocommerce-ios/assets/6242034/66739791-6b0d-406b-87b4-cfdcd76da20f)      | ![Simulator Screen Recording - iPhone 15 Pro - 2024-02-01 at 12 42 04](https://github.com/woocommerce/woocommerce-ios/assets/6242034/d7d86198-7f3c-4a89-a6ee-e04b56145a5b)       |
| ![Simulator Screen Recording - iPhone 15 Pro - 2024-02-01 at 12 44 01](https://github.com/woocommerce/woocommerce-ios/assets/6242034/dac44d5a-a5b9-46e1-9062-6c5112376e0c)      | ![Simulator Screen Recording - iPhone 15 Pro - 2024-02-01 at 12 42 28](https://github.com/woocommerce/woocommerce-ios/assets/6242034/0244463c-8b52-4677-9403-eb4632bfee88)       |

---
